client/quantum_serverless/quantum_serverless.py

```diff
@@ -144,6 +144,31 @@ def get_jobs(self, **kwargs):
         """
         return self._selected_provider.get_jobs(**kwargs)
 
+    def files(self):
+        """Returns list of available files to download.
+
+        Example:
+            >>> serverless = QuantumServerless()
+            >>> serverless.files()
+
+        Returns:
+            list of available files
+        """
+        return self._selected_provider.files()
+
+    def download(self, file: str, download_location: str = "./"):
+        """Downloads file.
```

Comment:
It may be better saying that the file down loaded in a different name and it's returned as the return value

Reply:
Agree! https://github.com/Qiskit-Extensions/quantum-serverless/pull/896/commits/84758f5a3c7f64a68fb1caad67c161ac1ecb8c7a